src/st_stuff.c

```diff
@@ -499,6 +502,31 @@ static boolean CheckConditions(sbarcondition_t *conditions, player_t *player)
                 result &= (!!cond->param == false);
                 break;
 
+            // Woof! additions
+            case sbc_healthgreaterequal:
+                result &= player->health >= cond->param;
+                break;
+
+            case sbc_ammogreaterequal_percent:
+                {
+                    ammotype_t type = weaponinfo[player->readyweapon].ammo;
+                    if (type != am_noammo)
+                    {
+                        result &=
+                            (player->ammo[type] * 100 / player->maxammo[type])
+                            >= cond->param;
+                    }
+                }
+                break;
+
+            case sbc_armorgreaterequal:
+                result &= player->armorpoints >= cond->param;
```

Comment:
I'd prefer color by armor type here.

Reply:
If this is getting too complicated, we could still change the `translation` value by function, right? Is it possible to define some kind of "callback function" for the status bar elements that gets called whenever they are rendered?

Reply:
I agree, fixed here: https://github.com/fabiangreffrath/woof/pull/1916/commits/5e954e257fa5dbc8dfc9ff63aa47264a4475d40a